Project/Source Files/Screens/Recipies/RecipiesTableViewController.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  RecipiesViewController.swift
+//  NetguruCollegeApp
+//
+
+import UIKit
+
+class RecipiesTableViewController: UIViewController {
+
+    @IBOutlet var tableView: UITableView!
+    var dataBase = [
+        Recipe(image: UIImage(named: "TestingImage")!,
+               title: "Let's cook something",
+               ingredients: "Garlic, beef, salt, Garlic, beef, salt, Garlic, beef, salt"),
+        Recipe(image: UIImage(named: "TestingImage2")!,
```

Comment:
For our purpose in the app currently it's completely fine, however, please remember, that generally using `!` is unsafe.